docs/downstreamdoc.yaml

```
@@ -0,0 +1,21 @@
+guides:
+  - src/main/asciidoc/datasource.adoc
+  - src/main/asciidoc/deploying-to-openshift.adoc
```

Please add one more... :pray: 

```suggestion
  - src/main/asciidoc/deploying-to-openshift.adoc
  - src/main/asciidoc/logging.adoc
```